### ISSUE TITLE
NoClassDefFoundError: Could not initialize class com.wynntils.core.managers.Managers Preventing Game Launch

### ISSUE THREAD

**Issue by san7890** [Wynntils/Wynntils#845]
Using Fabric. Found by compiling 246b9eed87e008ff8cb735a61b6c3de383c0a682 (most recent commit at the time of this issue). Appears to be a different bug than what that commit fixes (alpha version 170 crashes for a different reason, presumably because it's missing the code introduced in the aforementioned commit).

![image](https://user-images.githubusercontent.com/34697715/208035933-74ce0a30-a274-45a8-b7fa-f289cfe2b68f.png)

```txt
NoClassDefFoundError: Could not initialize class com.wynntils.core.managers.Managers

at com.wynntils.core.mod.CrashReportManager.generateDetails(CrashReportManager.java:35)
at net.minecraft.class_128.handler$bml000$addWynntilsDetails(class_128.java:525)
at net.minecraft.class_128.method_555(class_128.java:77)
at net.minecraft.class_128.method_568(class_128.java:132)
at net.minecraft.class_310.method_1565(class_310.java:842)
at net.minecraft.client.main.Main.main(Main.java:207)
at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:461)
at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:74)
at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23)
```

https://github.com/Wynntils/Artemis/blob/246b9eed87e008ff8cb735a61b6c3de383c0a682/common/src/main/java/com/wynntils/core/mod/CrashReportManager.java#L30-L42

**Comment by magicus** [comment]
@san7890 Can you please confirm that this was solved by #846.

**Comment by san7890** [comment]
Resolved, thank you!
